### DISCUSSION TITLE
Windows Setup

### DISCUSSION THREAD

**Discussion by LKPumngg** [kubilus1/autoortho#376] [Q&A] [answered]
### Please check existing docs and issues first.

- [x] I have checked for existing answers
- [X] This is not a bug or other "Issue"
- [X] I have read the FAQ and documentation

### System Details

- OS: Windows 10
- XPlane Version: XP12.05r1
- AutoOrtho Version 0.6.1
- Filesystem type NTFS

### question

In the instructions, I saw this passage, If both Dokan and WinFSP are installed, Dokan will be preferred. Users can force preferring WinFSP by setting prefer_winfsp=True in their config file, **but he said where in the configuration file can it be configured The document**

![image](https://github.com/kubilus1/autoortho/assets/142016467/c7a38ad2-52a8-4734-8848-4cd5d8a910d3)

**Comment by Jeff2122** [comment, marked as answer]
C:\Users\YOURUSERNAME\\.autoortho
Strange naming convention for most of us. It is a Linux carryover. Windows treats it as a hidden file.
                    .autoortho 
is what you are looking for.

**Reply by LKPumngg** [reply]
Is it this?

![image](https://github.com/kubilus1/autoortho/assets/142016467/5c7d1389-6490-4758-87e5-0e62ccef0806)

**Reply by Jeff2122** [reply]
You got it.
